src/build/mdn-comments.ts

```diff
@@ -117,37 +45,50 @@ function insertComment(
   }
 }
 
+function extractSlug(slug: string): string[] {
+  for (const subdirectory of subdirectories) {
+    if (!slug.toLowerCase().startsWith(subdirectory)) {
+      continue;
+    }
+    return slug
+      .slice(subdirectory.length)
+      .replace(/_static/g, "")
+      .split("/");
+  }
+  return [];
+}
+
+function generateComment(summary: string, name: string): string {
+  // Escape special regex characters in the name
+  const escapedName = name.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+
+  return summary
+    .replace(/\n/g, " ") // remove newlines
+    .replace(
+      // Match optional preceding identifier + dot OR just the name itself
+      new RegExp(`(?:\\b\\w+\\.)?${escapedName}(\\(\\))?`),
+      (match) => `**\`${match}\`**`,
```

Comment:
Would be happier with less regexp as possible, something like this should work:

```js
// `parent` can be `slugArr.at(-2)` from the caller
.replace(` ${parent}.${name} `, `**\`${$1}\`**`)
.replace(` ${name} `, `**\`${$1}\`**`)
```

Reply:
Honestly, I feel like we don't even need this. Was the style originally only this way because that's what the markdown in MDN did?

I'm going through the diff and there are places we are now _making_ bold that did not used to be. It seems like we'd save space and probably extra formatting noise by not bothering, and also avoid mistakenly causing issues with replacement.

Reply:
Example:

<img width="846" height="213" alt="image" src="https://github.com/user-attachments/assets/5a7ff9da-c816-4673-bf53-77ad65c3fa1e" />